ergotree-interpreter/src/eval/coll_append.rs

```diff
@@ -0,0 +1,73 @@
+use ergotree_ir::mir::coll_append::Append;
+// use ergotree_ir::mir::constant::TryExtractInto;
+use ergotree_ir::mir::value::Value;
+use ergotree_ir::mir::value::CollKind;
+
+
+use crate::eval::env::Env;
+use crate::eval::EvalContext;
+use crate::eval::EvalError;
+use crate::eval::Evaluable;
+
+fn _helper_concat<T>(mut e1 : Vec<T>, e2 : Vec<T>) -> Vec<T> {
```

Comment:
What is the reason for using the underscore prefix? Underscore prefix in variable/argument names silences the compiler warning for unused var.

Reply:
The python convention is to prefix every function not leaving the module/local scope with an underscore. I took this convention to rust even though rust has visibility modifiers (pub) that dont make it strictly necessary.

In my daywork we have this function in a util/grabbag file in the main project. Most rust internal functions like extend or iter.chain.collect are either bothersome or take a reference. Usually a reference is what you want but I like to consume both parts by taking ownership.

If the function remains a local helper you can name it whatever you like.